docker/lr-metrics/Makefile

```diff
@@ -1,16 +1,12 @@
-VERSION = 0.1.10
-TAG1 = quay.io/broad-long-read-pipelines/lr-metrics:$(VERSION)
-TAG2 = quay.io/broad-long-read-pipelines/lr-metrics:latest
-TAG3 = us.gcr.io/broad-dsp-lrma/lr-metrics:$(VERSION)
-TAG4 = us.gcr.io/broad-dsp-lrma/lr-metrics:latest
+VERSION = 0.1.11
```

Comment:
can you list which tasks use this version, in your PR?
This docker is used in 18 different tasks, I want to understand where it's touched.
Thanks.

Reply:
Right now only `ReadMetrics` uses this version.

Reply:
I see. Can we then bring all other tasks to `0.1.11`? The only meaningful change is the update to the custom GATK tool. It's unlikely other tasks are affected, but let's avoid fragmentation. Thoughts?

Reply:
That seems fine to me.  Updating all to `0.1.11`.